src/components/User/User.types.ts

```diff
@@ -1,6 +1,8 @@
 interface User {
-  id: string;
-  text: string;
+  id: number;
+  name: string;
+  username: string;
+  email: string;
 }
 
-export default User;
\ No newline at end of file
+export default User;
```

Comment:
`export interface User {...}` instead of `default`. This is not an error or something, but we're not exporting default(ly) interfaces. Most of the times you gonna have a file with a number of interfaces in there, all using named export.